ethers-core/src/types/crypto/keys.rs

```diff
@@ -286,20 +302,22 @@ impl<'de> Deserialize<'de> for PublicKey {
             where
                 S: SeqAccess<'de>,
             {
-                let mut bytes = [0u8; COMPRESSED_PUBLIC_KEY_SIZE];
+                let mut bytes = [0u8; FULL_PUBLIC_KEY_SIZE];
```

Comment:
We don't need this, and can stick to a 33-byte representation. Because there is a way to decompress into the uncompressed format:
https://docs.rs/elliptic-curve/0.6.0/elliptic_curve/sec1/struct.EncodedPoint.html#method.decompress